__mocks__/adobexd/sample_dump.json

```diff
@@ -0,0 +1 @@
+undefined
```

Comment:
No dump for Adobe XD? Might indeed be harder to do since the output is split up in multiple JSON files, could you write the result from the parser (before it gets mapped) to this JSON file?